src/index.html

```diff
@@ -8,7 +8,7 @@
       href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. Browsers cannot directly process SCSS files. You need to compile the SCSS into a CSS file and link to the resulting `.css` file instead.